deploy/Chart/templates/database/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: database
```

Comment:
Should this be more specific like `radius-db` or `radius-postgres`?

Reply:
This is fair :)

Reply:
I'm good with database. I think it's plenty clear. The namespace already says `radius-system`.